app/package.json

```
@@ -7,26 +7,26 @@
     "@aragon/api": "2.0.0-beta.8",
     "@aragon/api-react": "2.0.0-beta.8",
     "@aragon/ui": "1.0.0",
-    "@babel/polyfill": "^7.2.5",
+    "@babel/polyfill": "^7.7.0",
     "@githubprimer/octicons-react": "^8.5.0",
     "axios": "^0.19.0",
-    "codemirror": "^5.47.0",
+    "codemirror": "^6.0.0",
```

This needs to be fixed:
```suggestion
    "codemirror": "^5.50.0",
```